src/std/text.ab

```diff
@@ -32,21 +32,50 @@ pub fun replace_one(source, search, replace) {
     return result
 }
 
-/// Replaces all occurrences of a regex pattern in the content with the provided replace text.
+const SED_VERSION_UNKNOWN = 0
+const SED_VERSION_GNU = 1
+const SED_VERSION_BUSYBOX = 2
+const SED_VERSION_BSD = 3
```

Comment:
I declared these to use as "enum".
(Enum is proposed in a comment of #193, but not implemented)
I think this has better performance than string comparison.